README.md

```diff
@@ -72,3 +75,17 @@ require('babel').transform("code", {
   ],
 });
 ```
+
+For use with Babel outside of Ember, a `.babelrc.js` file will be needed:
```

Comment:
Generally speaking, this isn't something we officially support (using without ember-cli I mean). Can you tweak the verbiage here to make that clear?